.changeset/quick-melons-occur.md

```diff
@@ -0,0 +1,6 @@
+---
+'@keystonejs/fields': patch
```

Comment:
```suggestion
'@keystonejs/field-content': patch
```

Reply:
Boo, I'm bad at github, that's meant to be adding `field-content`, not replacing `fields` 🤷‍♂️

Reply:
Done.